configure.ac

```diff
@@ -5039,7 +5039,7 @@ AC_CHECK_DECLS([RTLD_LAZY, RTLD_NOW, RTLD_GLOBAL, RTLD_LOCAL, RTLD_NODELETE, RTL
 # determine what size digit to use for Python's longs
 AC_MSG_CHECKING([digit size for Python's longs])
 AC_ARG_ENABLE(big-digits,
-AS_HELP_STRING([--enable-big-digits@<:@=15|30@:>@],[use big digits (30 or 15 bits) for Python longs (default is system-dependent)]]),
```

Comment:
You have to update ``configure``, too. It's a trivial change, so manually editing is easier.

Reply:
Done.

Reply:
> Yes, I've spent the last hour trying to get `autoreconf` to not generate spurious changes. :-( I'll go for the manual edit option.

I have you covered: https://quay.io/repository/tiran/cpython_autoconf